tests/stress/stress_test.go

```diff
@@ -641,3 +642,64 @@ func TestSync_SubmitExtrinsicLoad(t *testing.T) {
 		// Output: Balance transferred from Alice to Bob: 100000000000000
 	}
 }
+
+func TestSecondarySlotProducer(t *testing.T) {
+	numNodes := 4
+	utils.Logger.Patch(log.SetLevel(log.Info))
+
+	// start block producing node first
+	node, err := utils.RunGossamer(t, numNodes-1,
+		utils.TestDir(t, utils.KeyList[numNodes-1]),
```

Comment:
Use `t.TempDir()` instead of `utils.TestDir`

Reply:
have changed the test